packages/binding-file/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "strict": true,
+        "strictFunctionTypes": true,
```

Comment:
I am doing some changes for other packages, is `strictFunctionTypes` required here? or is it sufficient to set `strict` to true?

Reply:
I think what I would like to see in the end is a _global_ tsconfig.json file used for all packages. 
Hence I took the same settings that are currently used in `core` to not getting any surprise later (and yes, `strictFunctionTypes `might not be required here )

https://github.com/eclipse-thingweb/node-wot/blob/b25d863ab85774570bf203e126670d2975a0aa77/packages/core/tsconfig.json#L4-L5

Reply:
Checked the [doc](https://www.typescriptlang.org/tsconfig/#strict):

> Turning this on is equivalent to **enabling all** of the strict mode family options, which are outlined below.

So yeah in a followup PR we should remove this one here and the one in core (or everywhere else).